### PR TITLE
feat: [SRM-14413]: made changes to SLO listing page to show downtime active status tooltip

### PR DESCRIPTION
**Pull request by karansaraswat19**
#### Screenshots
Screenshot 2023-03-31 at 5 48 29 PM

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by karansaraswat19** [comment]
retrigger jest

**Comment by karansaraswat19** [comment]
retrigger jest

File: src/modules/85-cv/pages/slos/components/SLOActions/SLOActions.tsx [hunk 1]
```diff
@@ -38,12 +43,36 @@ export default function SLOActions(props: SLOActionsProps) {
     }
   })
 
+  const RenderTooltip = (): JSX.Element => {
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/slos/components/SLOActions/SLOActions.tsx:46]: Could be move as separate file

File: src/modules/85-cv/pages/slos/components/SLOActions/SLOActions.module.scss [hunk 2]
```diff
@@ -1,7 +1,29 @@
...
 }
+
+.downtimeIcon {
+  padding: var(--spacing-xsmall);
+}
+
+.tooltip {
+  p {
+    color: var(--grey-0) !important;
+    font-size: 12px !important;
```
Comment (praneshg239) [inline src/modules/85-cv/pages/slos/components/SLOActions/SLOActions.module.scss:23]: Use font size from theme.

```css
  --font-size-xsmall: 10px; /* very small text */
  --font-size-small: 12px; /* H4, small text */
  --font-size-normal: 14px; /* H3, normal text */
  --font-size-medium: 18px; /* H2 */
  --font-size-large: 28px; /* formerly 29px; need to be an even number */
  --font-size-xlarge: 32px;
```

File: src/modules/85-cv/pages/slos/components/SLOActions/SLOActions.tsx (hunk 1 above)
Comment (praneshg239) [reply src/modules/85-cv/pages/slos/components/SLOActions/SLOActions.tsx:46]: Agree.

Comment (harshilbits) [reply src/modules/85-cv/pages/slos/components/SLOActions/SLOActions.tsx:46]: Agree

**Review by harshilbits — APPROVED** [review]

**Pull request merged**